src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -10,7 +10,9 @@
 }
 
 .telescope-post-content p {
-  margin: 1.2rem 0;
+  padding: 0;
+  margin: 0;
+  display: inline;
```

Comment:
These changes on `<p>` break how paragraphs show-up in general.  A paragraph should be a block level element, and have vertical space above and below (left is dev, right is this PR):

<img width="1557" alt="Screen Shot 2020-04-25 at 8 52 22 PM" src="https://user-images.githubusercontent.com/427398/80294378-c9180600-8736-11ea-96ad-57e533443a00.png">

Reply:
@humphd I haven't realize I applied changes to the `<p>` element. Probably was trying to remove extra space and forgot to remove changes. Fixing it right now